src/segment_winreg.go

```diff
@@ -62,3 +74,45 @@ func (wr *winreg) templateString(segmentTemplate string) string {
 	}
 	return text
 }
+
+func (wr winreg) GetRegistryString(path string) (string, error) {
+	regValue, err := wr.env.getWindowsRegistryKeyValue(path)
+
+	if regValue != nil {
```

Comment:
Inverting this if allows the send if to be on the same level below.

Reply:
ooh, good spot - yeah, makes it a lot cleaner 👍